### ISSUE TITLE
authorinitials cannot be explicitly set

### ISSUE THREAD

**Issue by graphitefriction** [asciidoctor/asciidoctor#669]
When I explicitly set the `authorinitials` attribute, it is ignored.

Example:

```
= The Dangerous and Thrilling Documentation Chronicles
:author: Kismet Rainbow Chameleon 
:email: kismet@asciidoctor.org
:authorinitials: K.C.

This journey begins...

== About the Author

You can contact {author} at {email}.
{firstname} loves to hear from other chroniclers.

P.S. And yes, my middle name really is {middlename}.
What else would you expect from a member of the Rocky Mountain {lastname} Clan?

{authorinitials}
```

As you can see in the image below, the `authorinitials` is rendered as _KRC_ instead of _K.C._

![author-email-long](https://f.cloud.github.com/assets/1124042/1256416/d9d6ee48-2b94-11e3-9a5c-4fead77905d9.png)

**Comment by Mogztter** [comment]
We could also support a shorter syntax (inspired by https://github.com/asciidoctor/asciidoctor/issues/223)

```
Doc Writer <doc.writer@asciidoc.org>; Kismet Rainbow Chameleon <kismet@asciidoctor.org>
:authorinitials: DW; K.C.
```
 
Or even create a syntax to override initials in the author line:

```
Doc Writer (DW) <doc.writer@asciidoc.org>; Kismet Rainbow Chameleon (K.C.) <kismet@asciidoctor.org>
```

**Comment by mojavelinux** [comment]
I think the original idea behind author initials is that they are a calculated value, hence why the explicit assignment is ignored. However, I've already seen cases where this breaks down and an explicit assignment is needed.

I agree the first option makes a lot of sense. If we proceed with allowing the explicit assignment of author initials, that's definitely how we should handle multiple authors.

The second option is interesting. However, we should be careful before we use the bracket syntax as that could be a valuable space for another field. For instance, that might be a good place to put the author's title. So before we take it, we should think hard about what's more important. I'd like to see examples of other shorthand formats for author information.

**Comment by mojavelinux** [comment]
> I'd like to see examples of other shorthand formats for author information.

One place to look are bibliography formats.

**Comment by mojavelinux** [comment]
One possible way to do initials are as follows:

```
Kismet Rainbow Chameleon. KC. <Kismet Rainbow Chameleon>
```

The period after both the name and the initials indicates that it's a set of initials.

That way, we can still do title like this:

```
Kismet Rainbow Chameleon. KC. (Project Lead) <Kismet Rainbow Chameleon>
```

Just an idea.

**Comment by mojavelinux** [comment]
Alternatively, we could put periods on either side of the initials:

```
Kismet Rainbow Chameleon .KC. <Kismet Rainbow Chameleon>
```

**Comment by mojavelinux** [comment]
...or perhaps hyphens:

```
Kismet Rainbow Chameleon -KC- <Kismet Rainbow Chameleon>
```

**Comment by mojavelinux** [comment]
The hyphens really look nice, I think. And they aren't easily confused.

**Comment by mojavelinux** [comment]
...one more suggestion, square brackets for initials.

```
Kismet Rainbow Chameleon [KC] <Kismet Rainbow Chameleon>
```

Vote away!

**Comment by Mogztter** [comment]
Parentheses for title work great. Hyphens for initials looks nice but not with composite names like Jean-Claude.
Square brackets is maybe better in that case (as squared brackets are guaranteed to not be included in the initials):
```
Jean-Claude Van Damme [J-C VD] (Actor) <jcvd@asciidoctor.org>
```
(we could argue that Jean-Claude Van Damme initials are `JCVD` and not `J-C VD`)

As opposed to:
```
Jean-Claude Van Damme -J-C VD- (Actor) <jcvd@asciidoctor.org>
```

**Comment by mojavelinux** [comment]
On a second look, I like the first format, but I don't dislike the second. Good example, btw!

**Comment by mojavelinux** [comment]
I've opened a separate issue to cover the discussion that emerged from this issue. See #2942.

It's possible to override the authorinitials if the author is defined using the implicit author line.

```
= The Dangerous and Thrilling Documentation Chronicles
Kismet Rainbow Chameleon <kismet@asciidoctor.org>
:authorinitials: K.C.
```

It turns out to be really hard to support this override otherwise.

**Comment by mojavelinux** [comment]
Should we just treat this as a documentation issue?

**Comment by mojavelinux** [comment]
I found a way to support this specific case. See #2943. However, the fix breaks down when multiple authors are specified using the `authors` attribute. I think we could merge this fix for 1.5.8 since it's a pretty minor change, but I think for 2.0.0 we need to take as step back and rethink how authors are processes.
